src/Akka.Hosting/LoggerConfigBuilder.cs

```diff
@@ -96,6 +98,13 @@ public LoggerConfigBuilder ClearLoggers()
             return this;
         }
 
+        public LoggerConfigBuilder WithLogFilter(Action<LogFilterBuilder> filterBuilder)
```

Comment:
One thing I'm not so sure about - what happens if someone calls this and `ConfigureLogging` (or whatever it's called?) - can't we end up with one config overriding the other?

Reply:
Yes, the two implementation inside `ConfigureLoggers(builder)` and the extension method `AkkaConfigurationBuilder.WithLogFilter()` is exclusive to each other. The first being added would win because of how we retrieve `Setup` classes in `ActorSystem.Settings`

Reply:
LGTM - we decided to stick with just the `LogConfigBuilder`.